### ISSUE TITLE
question about vgg nomalization.

### ISSUE THREAD

**Issue by dypark86** [naoto0804/pytorch-AdaIN#13]
thanks for your works!

i have a question about nomalization.

vgg network normalize with imagenet RGB mean std value.
but i don't find code of vgg nomalization.

Is there reason not to use vgg nomalization?
 - pretrained network nomalize with other method?

**Comment by naoto0804** [comment]
Thanks for your interest! 
I was just following the [original implementation](https://github.com/xunhuang1995/AdaIN-style).
They also didn't use vgg normalization as you can see.
